docs/index.rst

```diff
@@ -56,10 +56,15 @@ variety of platforms.
 
     cloud/emr
 
+.. toctree::
+    :caption: Demo notebooks
+    :maxdepth: 2
+
+    notebooks/mango-python-alignment.ipynb
+    notebooks/mango-python-coverage.ipynb
 
 
-* :ref:`genindex`
```

Comment:
why is genindex removed?

Reply:
It seemed to go to unpopulated index and was causing a strange line break